src/Tools/BuildUtil/FindNuGetAssetsForVsix.cs

```diff
@@ -5,6 +5,8 @@
 using Microsoft.Build.Framework;
 using Microsoft.Build.Utilities;
 
+#pragma warning disable CA1819 // CA1819: Properties should not return arrays 
```

Comment:
Not sure if we should have some special case for build tasks -- this is a _very_ common pattern there.